ledger/metrics/src/main/scala/com/daml/metrics/OpenTelemetryMeterOwner.scala

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import com.daml.ledger.resources.{Resource, ResourceContext, ResourceOwner}
+import com.daml.metrics.MetricsReporter.Prometheus
+import io.opentelemetry.api.metrics.Meter
+import io.opentelemetry.exporter.prometheus.PrometheusCollector
+import io.opentelemetry.sdk.metrics.SdkMeterProvider
+
+import scala.annotation.nowarn
+import scala.concurrent.Future
+
+@nowarn("msg=deprecated")
```

Comment:
Could you add some text explaining why we are introducing a deprecated class.  Note that [PrometheusCollector](https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/prometheus/src/integrationTest/java/io/opentelemetry/exporter/prometheus/PrometheusCollector.java) exists in `integrationTest` and is also deprecated.

Reply:
Good point, will add to the code, it was initially documented in the ticket itself.

Reply:
Added a comment about the usage of the deprecated class